.github/workflows/testing.yml

```diff
@@ -38,12 +40,11 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install flake8 nbconvert[test]
+        pip install nbconvert[test]
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+        pip install git+https://github.com/AlexandreHutton/stroke
     - name: Test
       run: |
-        flake8 *.py submissions/*/*.py
         python download_data.py
         ramp-test --quick-test
         ramp-test --submission sample --quick-test
-        jupyter nbconvert --execute stroke_starting_kit.ipynb --to html --ExecutePreprocessor.kernel_name=python3
```

Comment:
@AlexandreHutton these CI scripts are almost identical for all our kits. If possible I would prefer you don't edit them too much. And typically using flake8 is a good thing :)

Reply:
I restored the Notebook -> html conversion, but I don't think that flake is providing anything useful (especially with the 80-column restriction). The ratio of additional required time to return is not in favour of flake, so I've opted to remove it.

Reply:
you can add a setup.cfg to the repo to disable certain flake8 errors you want to ignore.

see an example

https://github.com/braindecode/braindecode/blob/master/setup.cfg#L17